pacoloco.go

```diff
@@ -132,10 +146,27 @@ func main() {
 	log.Println("Starting server at port", config.Port)
 	// The request path looks like '/repo/$reponame/$pathatmirror'
 	http.HandleFunc("/repo/", pacolocoHandler)
-	// http.HandleFunc("/stats", statsHandler) TODO: implement stats
+	// Expose prometheus metrics
+	http.Handle("/metrics", promhttp.Handler())
 	log.Fatal(http.ListenAndServe(listenAddr, nil))
 }
 
+func gatherCacheSizeAndNumOfPackages(repoDir string) (float64, float64, error) {
```

Comment:
A nitpick, maybe rename the method to `gatherCacheStats` and then add documenting comments on what the function returns? You can also name the return parameters for clarity.